Art/rajan4405/app.css

```diff
@@ -0,0 +1,39 @@
+.circle {
+  width: 200px;
+  height: 200px;
+  background-color: magenta;
+  transition: background-color 1s, border-radius 2s;
+  /* Transition: PropertyName, Duration, Timing Function, Delay */

```

Comment:
# Feedbacks 
- [ ] Unnecessary comment that could be removed
Please remove the comment

Reply:
Already removed unnecessary comment from the app.css file